_overviews/tutorials/scala-on-android.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: singlepage-overview
+title: A Tutorial on writing Scala apps on Android
+
+partof: scala-on-android
+
+permalink: /tutorials/:title.html
+---
+
+By Maciej Gorywoda
+
+## Introduction
+The Android platform runs on Android Runtime which is a virtual machine based on JVM and, although not identical,[it's very similar to it](https://www.baeldung.com/java-jvm-vs-dvm). It's no surprise then that two of the most popular programming languages used to create Android apps are Java and Kotlin, both JVM languages. It might be a bit surprising though that Scala, also a JVM language and one in popularity [on par or ahead of Kotlin](https://redmonk.com/sogrady/2021/03/01/language-rankings-1-21/), has little presence on the Android platform. But it is possible to write Android apps in Scala, and in fact it's possible to do it in more than one way. Here, in this document, we will focus on how to write a modern Android app with Scala that uses GraalVM Native Image and JavaFX. At the end of this tutorial you will links to materials discussing other options. 
+
+## How to build an Android app with GraalVM Native Image 
+
+#### Requirements
```

Comment:
You should add that we will use `mvn`.

Reply:
Done. I just wrote "For building, we will use Maven". I guess people should know what it is.